doc/Language/faq.pod6

```diff
@@ -370,6 +370,99 @@ L<.comb|/routine/comb> is accepting an optional L<Int|/type/Int>:
 .say for 'abcdefghijklmnopqrstuvwxyz'.comb: 8;  # OUTPUT: «abcdefgh␤ijklmnop␤qrstuvwx␤yz»
 =end code
 
+=head2 List: How can I remove duplicates items in a list
+
```

Comment:
"duplicate", also question mark at the end.
And... do you really think this is a FAQ? Woudln't it be better served by simply adding more information to the [existing description of `unique`](https://docs.raku.org/routine/unique)

Reply:
Removed.

I am sure it is one of the 10 most frequent questions on code operation. In the top 10 view for python and top 15 for [algorithm](https://stackoverflow.com/questions/tagged/algorithm?sort=MostVotes&edited=true)